GitExtUtils/FileLogging/Logger.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace GitExtUtils.FileLogging
+{
+    /// <summary>
+    /// For unit testing
+    /// </summary>
+    public class LoggerStub : ILogger
```

Comment:
Im getting lost here... 
Why do we have a unit testing specific class in the prod codebase? Why is it public?

Reply:
Because for as long as the static logger lives, we need an easy way to fixup all the unit tests. It is public to ensure that you can use it from all the unit tests if you want to assert a certain logging.

Perhaps the same could be achieved with static initializers in each test project. But they still need to share the definition. Is there a better assembly that all the tests projects depend on? We could place it there